### ISSUE TITLE
In-game music

### ISSUE THREAD

**Issue by grady404** [cooper-anderson/Blockade#56]
So here's my sample track for Blockade's in-game music. Tell me if you like the style of it, or if you think it should be more tranquil/relaxing than it currently is. Be sure to listen with headphones if you're able to.

[Blockade Music - Sample.mp3.zip](https://github.com/cooper-anderson/Blockade/files/477941/Blockade.Music.-.Sample.mp3.zip)

**EDIT:** Follow all soundtrack updates [here](https://soundcloud.com/gradyshoemaker/sets/blockade-ost/s-3gb5O).

**Comment by grady404** [comment]
Oh yeah, listen with headphones if you can. (I edited the post)

**Comment by cooperuser** [comment]
Music sounds cool, but I think that it is a bit too complex for the game, I was thinking maybe less stuff happening? sort of like what Besiege did with how you sometimes here sounds come and go.

**Comment by grady404** [comment]
Sure, I can make something more relaxing if you want

**Comment by cooperuser** [comment]
Awesome

**Comment by grady404** [comment]
Oh and, for the sound effect that plays when you finish a level, do you want it to be more of a "chime" (which might clash with the music but not if they're in a similar key), or a sound more similar to a drum hit or soft tone (see N++'s level win sound)?

**Comment by grady404** [comment]
[This](https://www.youtube.com/watch?v=QPvy05vzRmY&t=50s) is what I meant by a "soft tone"

**Comment by cooperuser** [comment]
Hmm, not sure

**Comment by grady404** [comment]
Do you want any sound when a block initiates movement? Or only when it finishes sliding?

**Comment by cooperuser** [comment]
Maybe a small *thud* when it hits a wall

**Comment by grady404** [comment]
But nothing when it starts moving?

**Comment by cooperuser** [comment]
Well, the moving sound will start

**Comment by grady404** [comment]
Well yeah, so no then?

**Comment by cooperuser** [comment]
Yeah, no sound on start

**Comment by grady404** [comment]
Alright, here's my second attempt at in-game music. It's a lot more mellow than my last track (and I think better, too), but still not nearly as mellow as Besiege's music (which is probably a good thing to be honest, since I didn't find that game's music to be memorable at all). Tell me what you think.

[Blockade Music B.mp3.zip](https://github.com/cooper-anderson/Blockade/files/483883/Blockade.Music.B.mp3.zip)

**Comment by cooperuser** [comment]
Awesome, I'm not at my computer right now so I couldn't listen to it that well, how well does it loop?

**Comment by grady404** [comment]
I _could_ loop it, but I wasn't planning on it. I was going to shoot for the music flow of N++, where each song is about five to eight minutes long (estimated) and it works like a regular music playlist, where you can choose to play the songs in order or shuffle them.

**Comment by cooperuser** [comment]
Ah, I see.. But I think that would add unnecessary complexity to the settings

**Comment by grady404** [comment]
Have you seen N++'s settings? Don't talk about complexity until you see that shit. But on a separate note, N++'s music controls are completely separate from the settings, if you press and hold left trigger and right trigger at the same time, an interface comes up where you can select a song from a list. I think we should implement something like that.

**Comment by cooperuser** [comment]
Oh, yeah I guess that makes sense

**Comment by grady404** [comment]
Same thing goes for themes, a la N++

**Comment by grady404** [comment]
Just uploaded the finished (finished, but not _finished_) in-game music [here](https://soundcloud.com/gradyshoemaker/blockade-ost-1/s-PY9nu) on my SoundCloud page (which I literally just created). Let me know if there's anything you think I should change, add, or get rid of to make the track better/more interesting.

**Comment by cooperuser** [comment]
Awesome, I shall listen to it now.

**Comment by grady404** [comment]
I recommend using headphones.
- [ ] will use them

**Comment by cooperuser** [comment]
- [x] Am currently using them
- [x] ok good

**Comment by cooperuser** [comment]
Just finished listening to it, I think its great, but would it sound bad if it were slowed down a little bit?

**Comment by grady404** [comment]
I can mess around with the tempo and see how it sounds. Anything else I should change? Are there any parts that you think I should try to add more to?

**Comment by cooperuser** [comment]
No, I like the simplicity

**Comment by grady404** [comment]
So you want the tempo slowed down just a "little bit" or, like, a fair amount? It's at 85 bpm in the version I posted. I tried 82 bpm and 80 bpm. 82 bpm still sounds fast enough to keep a strong beat going, without giving the feeling that it is going too fast. 80 bpm sounds significantly slower and more relaxing. I can replace the file on SoundCloud with one of the two versions, and if you don't like the tempo I can try the other one. Which one should I go with first?

**Comment by grady404** [comment]
As you requested, I have slowed down the tempo of the song, and I also completely reworked the second half. [Here](https://soundcloud.com/gradyshoemaker/blockade-ost-1/s-JTXZZ) it is, tell me what you think.

**Comment by cooperuser** [comment]
Listening to it now

**Comment by grady404** [comment]
Also, another thing is, in the older version the main melody would start to seem repetitive/get on my nerves near the end of the song if I listened all the way through. But luckily that doesn't seem to happen anymore, maybe it has something to do with drowning that tune out with the other instruments or something?

**Comment by cooperuser** [comment]
Not sure, But I think I understand what you are saying

**Comment by grady404** [comment]
Tempo sound good, or should I drop it down one more notch?

**Comment by cooperuser** [comment]
Yeah, I think it should still be at least a tiny bit slower, and quieter, but I think the volume can be controlled through the programming.

**Comment by grady404** [comment]
Quieter, as in actually just making the entire song quieter, or is one (or more than one) of the instruments too overpowering? And I'll update the tempo and repost the link right now.

**Comment by cooperuser** [comment]
Quieter as in turning town the master volume, which is always re-adjustable, and even the user can adjust it.

**Comment by grady404** [comment]
So the mixing is fine?

**Comment by cooperuser** [comment]
Yeah

**Comment by grady404** [comment]
Finished slowing the tempo down and changing a couple more things to make it sound nicer. I made a playlist [here](https://soundcloud.com/gradyshoemaker/sets/blockade-ost/s-3gb5O) (also up at the top of this thread), so I shouldn't have to keep reposting the link.
